common/list/_lists.scss

```
@@ -73,7 +74,71 @@ ul[class*=list-col] {
 	}
 }
 
+/*
+ * this code is fully compatible with the equivalent CSS
+ * in the wet-boew code here: https://github.com/wet-boew/wet-boew/blob/master/src/base/lists/_base.scss
+*/
 ul.list-col-xs-1 > li { flex-basis: 100% }
 ul.list-col-xs-2 > li { flex-basis: 50% }
 ul.list-col-xs-3 > li { flex-basis: 33.33% }
 ul.list-col-xs-4 > li { flex-basis: 25% }
+
+
+div.lst-lwr-alph>ol,
+.lst-lwr-alph {
+	list-style-type: lower-alpha;
+}
+
+div.lst-upr-alph>ol,
+.lst-upr-alph {
+	list-style-type: upper-alpha;
+}
+
+div.lst-lwr-rmn>ol,
+.lst-lwr-rmn {
+	list-style-type: lower-roman;
+}
+
+div.lst-upr-rmn>ol,
+.lst-upr-rmn {
+	list-style-type: upper-roman;
+}
+
+.lst-num {
+	list-style-type: decimal;
+}
+
+div.lst-none>ul,
+.lst-none {
+	list-style-type: none;
+}
+
+div.lst-spcd>ul>li,
+div.lst-spcd>ol>li,
+.lst-spcd>li {
+	margin-bottom: 10px;
+}
+
+/*
+ * The CSS for list-unstyled and list-inline classes is adapted from
+ * https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/css/theme.min.css
+ * to address the specific scenario where these styles are applied to a parent div.
+ * This adaptation accommodates edge cases where the styles need to be inherited by the parent element.
+ */
```

Add the following comment

```suggestion
 * This adaptation accommodates edge cases where the styles need to be inherited by the parent element.
 *
 * Additional style for list also defined here: https://github.com/twbs/bootstrap-sass/blob/v3.4.3/assets/stylesheets/bootstrap/_type.scss#L146 
 */
```